src/main/java/frc/team2412/robot/commands/limelight/GetWithinDistanceCommand.java

```diff
@@ -0,0 +1,43 @@
+/*package frc.team2412.robot.commands.limelight;
+
+
+public class GetWithinDistanceCommand extends Command {
+
+	private LimelightSubsystem limelightSubsystem;
+	private DrivebaseSubsystem drivebaseSubsystem;
+	private Pose2d currentPose;
+
+	private Command getToPositionCommand;
+
+	public GetWithinDistanceCommand(
+			LimelightSubsystem limelightSubsystem, DrivebaseSubsystem drivebaseSubsystem) {
+		this.limelightSubsystem = limelightSubsystem;
+		this.drivebaseSubsystem = drivebaseSubsystem;
+
+		// find target -> get current pose -> use current pose and target to calculate target pose ->
+		// move towards target pose
+
```

Comment:
Need to require the drivebaseSubsystem so nothing else tries to drive while this command is running

Reply:
The whole GetWithinDistanceCommand.java file is redundant and has been removed